native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/DefaultStatementParameterProcessor.java

```diff
@@ -821,7 +824,15 @@ private void setDateTimeAndTimestamp(PreparedStatement preparedStatement, String
                     } else {
                         preparedStatement.setTimestamp(index, Timestamp.valueOf(localDateTime));
                     }
-                } catch (Throwable e) {
+                } catch (Throwable ex) {
+                    if (ex instanceof SQLFeatureNotSupportedException) {
+                        if (zoneMap != null) {
+                            throw new TypeMismatchError("Unsupported type: offset of Civil is not supported by " +
+                                    "this database. " + ex.getMessage());
```

Comment:
```suggestion
                                    "this database." + ex.getMessage());
```

Reply:
We need to add the space. Otherwise, the error message will follow this msg without space.